thunderbird/processes/wps_update_metadata.py

```diff
@@ -27,21 +28,13 @@ class UpdateMetadata(Process):
     def __init__(self):
 
         inputs = [
-            ComplexInput(
-                "opendap",
-                "Daily NetCDF Dataset",
-                abstract="Path to OpenDAP resource",
-                min_occurs=0,
-                max_occurs=1,
-                supported_formats=[FORMATS.DODS],
-            ),
             ComplexInput(
                 "netcdf",
                 "Daily NetCDF Dataset",
                 abstract="NetCDF file",
-                min_occurs=0,
+                min_occurs=1,
                 max_occurs=1,
-                supported_formats=[FORMATS.NETCDF],
+                supported_formats=[FORMATS.NETCDF, FORMATS.DODS],
             ),
```

Comment:
Nicely working! However, little changes needed in jupyter lab since we do not have `opendap` keyword anymore.